CHANGELOG

```diff
@@ -40,6 +40,11 @@ snowplow-web 0.14.0 (2023-03-28)
 ## Summary
 This version contains two major changes, the first is to migrate our models away from the `snowplow_incremental_materialization` and instead move to using the built-in `incremental` with an optimization applied on top. The second is to change the de-duplication logic applied to redshift/postgres to bring it in line with the other warehouses (keeping 1 of the duplicate records, instead of discarding them all). We also upgrade some macros and update some of our docs.
 
+snowplow-web 0.14.0 (2023-03-28)
+---------------------------------------
+## Summary
+This version contains two major changes, the first is to migrate our models away from the `snowplow_incremental_materialization` and instead move to using the built-in `incremental` with an optimization applied on top. The second is to change the de-duplication logic applied to redshift/postgres to bring it in line with the other warehouses (keeping 1 of the duplicate records, instead of discarding them all). We also upgrade some macros and update some of our docs.
+
```

Comment:
Yes, the result of multiple rebase. I think it will get corrected once it is merged to the release branch. I will rebase the commits to keep two before merge.

Reply:
Looking at the current commit history I think it is safer to just squash and merge as core web vitals and remaining ga4 fields in one commit.